zsl/task/job_context.py

```diff
@@ -6,29 +6,63 @@
 """
 from __future__ import unicode_literals
 from builtins import object
+from typing import Callable
+from abc import abstractmethod
+
 from zsl.task.task_data import TaskData
 from zsl.application.service_application import service_application
-from abc import abstractmethod
 
 
 class Job(object):
     def __init__(self, data):
+        # type: (dict) -> None
+
         self.data = data
 
+    @property
+    def path(self):
+        """Job's path.
+
+        :getter: Returns job's path
+        :type: str
+        """
+        return self.data['path']
+
+    @property
+    def payload(self):
+        """Data part of job.
+
+        :getter: Returns job's payload
+        :type: dict
+        """
+        return self.data['data']
+
+    @property
+    def is_valid(self):
+        """Validity of job's data.
+
+        :getter: Returns if job's data are valid
+        :type: bool
+        """
+        return self.data and 'path' in self.data and 'data' in self.data
+
 
 class JobContext(object):
-    """
-    Job Context
-    """
+    """Job Context"""
+
+    _current_context = None
```

Comment:
The `_current_context` was initialized in the `__init__` as instance variable, but then with get/set_current_context it was assigned as a class variable?